Package.swift

```diff
@@ -117,3 +117,9 @@ let package = Package(
   dependencies: dependencies,
   targets: targets
 )
+
+for target in package.targets {
+  var settings = target.swiftSettings ?? []
+  settings.append(.enableUpcomingFeature("MemberImportVisibility"))
+  target.swiftSettings = settings
+}
```

Comment:
Can we just add this directly to the `defaultSwiftSettings` above so that they're all in the same place.

Reply:
The intention here is that many of the core Swift on Server repositories have a block of their package manifest which a common structure which can be easily be modified with a script. This block would be used to insert settings which we wish to apply across all repositories.